server/courses/el/eisagwgh-sth-dhmosiografia-dedomenwn/index.yaml

```diff
@@ -0,0 +1,26 @@
+title: Introduction to Data Journalism
+introduction_html: |-
+    <p>
+    This entry level course will teach you how to answer questions with data in a transparent
+    and reproducible manner, without the need to code.
+    </br></br>
```

Comment:
Should be `</p><p>`. (The English one should certainly be `</p><p>`, too. I just fixed the English one. I have no patience for invalid HTML.)